### PR TITLE
add mobile nav styling

### PR DESCRIPTION
**Pull request by learykara**
image

image

### CODE REVIEW

File: src/AppBar/Navigation.jsx [hunk 1]
```diff
@@ -55,69 +53,70 @@ class AppNav extends React.Component {
...
-                <NavLink className="app-nav-link__in-menu" {...link} />
-              </li>
-            ))}
+            <div className="app-nav-list-sub">
+              {linkSet.menu.map(link => (
+                <li key={uniqueId('link_')}>
+                  <NavLink className="app-nav-link__in-menu" {...link} />
+                </li>
+              ))}
+            </div>
```
Comment (ripleyaffect) [inline src/AppBar/Navigation.jsx]: This component can be broken up into several components

File: src/Logo/BlocLogo.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+);
+
+BlocLogo.propTypes = {
+  color: PropTypes.string,
+  size: PropTypes.number,
+};
+
+BlocLogo.defaultProps = {
+  color: '#3E2AEE',
+  size: 24,
```
Comment (ripleyaffect) [inline src/Logo/BlocLogo.jsx]: oh I see now you using size

File: src/Logo/Logo.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const Logo = ({ brand, ...props }) =>
+  brand === 'bloc' ? <BlocLogo {...props} /> : <TFLogo {...props} />;
+
+Logo.propTypes = {
+  brand: PropTypes.string,
+};
+
+Logo.defaultProps = {
+  brand: null,
```
Comment (ripleyaffect) [inline src/Logo/Logo.jsx]: not `thinkful`?

File: src/Logo/BlocLogo.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  </svg>
+);
+
+BlocLogo.propTypes = {
+  color: PropTypes.string,
+  size: PropTypes.number,
+};
+
+BlocLogo.defaultProps = {
+  color: '#3E2AEE',
```
Comment (ripleyaffect) [inline src/Logo/BlocLogo.jsx]: are the vars not anywhere here?

Comment (learykara) [reply src/Logo/BlocLogo.jsx]: how would I pull a CSS var into JS?

Comment (ripleyaffect) [reply src/Logo/BlocLogo.jsx]: https://www.npmjs.com/package/less-vars-to-js looks promising

**Pull request merged**